### ISSUE TITLE
Occasional regressions in performance tests

### ISSUE THREAD

**Issue by vtomole** [quantumlib/Cirq#1813]
When re-run, these tests usually pass. We should increase the margin of error for these checks.

See: https://github.com/quantumlib/Cirq/pull/1796#issuecomment-511494180

Update: https://travis-ci.com/quantumlib/Cirq/jobs/215339790 which is related to https://github.com/quantumlib/Cirq/issues/1804 which was probably caused by https://github.com/quantumlib/Cirq/pull/1752 might be what is slowing down this performance test even if it's for a tiny bit.

**Comment by Strilanc** [comment]
I disagree that we should increase the margin of error. I think these periodic failures being ignored indicates that these tests are not fulfilling their function.

Since the tests are not being respected, and I don't see a viable way to force them to be respected in a way that doesn't put undue burden on unrelated contributions, I think we should remove the performance tests.

If we want performance testing, I think it has to be a sort of historical affair where someone clones the repository then runs the performance tests over a large number of versions in order to produce a plot. They would then try to make that plot better.

In other words, it doesn't make sense for performance to block submission unless we're seeing a performance regression that's large enough to actually block on (like a 2x slowdown).
